app/Helpers/render/site-award.php

```diff
@@ -197,6 +199,39 @@ function RenderAward(array $award, int $imageSize, string $ownerUsername, bool $
         return;
     }
 
+    if ($awardType == AwardType::Event) {
+        $event = Event::find($awardData);
+        if ($event) {
+            $tooltip = "Awarded for completing the {$event->title} event";
+            $image = $event->image_asset_path;
+
+            if ($awardDataExtra !== 0) {
+                $eventAward = EventAward::where('event_id', $awardData)
+                    ->where('tier_index', $awardDataExtra)
+                    ->first();
+
+                if ($eventAward) {
+                    $image = $eventAward->image_asset_path;
+
+                    if ($eventAward->achievements_required < $event->game->achievements_published) {
+                        $tooltip = "Awarded for earning at least {$eventAward->achievements_required} achievements in the {$event->title} event";
+                    }
+                }
+            }
+
+            echo avatar('event', $event->id,
+                link: route('game.show', $event->game->id),
+                tooltip: "<div class='px-2 py-2'><span>$tooltip</span><p><span>{$awardDate}</span></div>",
```

Comment:
Not overly concerned with UX at the moment - this will all get migrated to React later anyway. This may be a marginal improvement though:
```php
                tooltip: "<div class='p-2 max-w-[320px] text-pretty'><span>{$tooltip}</span><p class='italic'>{$awardDate}</p></div>",
```
Changes:
* Combines "px-2 py-2" into "p-2" (p-* applies padding on all 4 sides).
* Adds a nominal max width and text balancing class.
* Fixes an invalid HTML issue (missing a closing p tag).
* Italicizes the date.

![Screenshot 2025-01-16 at 9 21 49 PM](https://github.com/user-attachments/assets/25846302-5591-4696-ab42-48f6e25d5192)

We can take some of these, all of these, or none of these. Just suggestions!

Reply:
Applied suggested changes. Also remove timestamp from date (it doesn't appear in any other badge tooltips)